tests/nvapi_system.cpp

```diff
@@ -203,6 +205,13 @@ TEST_CASE("Sysinfo methods succeed against local system", "[system]") {
         result == NVAPI_OK && frequencies.domain[NVAPI_GPU_PUBLIC_CLOCK_VIDEO].bIsPresent
             ? std::cout << std::dec << frequencies.domain[NVAPI_GPU_PUBLIC_CLOCK_VIDEO].frequency / 1000 << "MHz" << std::endl
             : std::cout << "N/A" << std::endl;
+
+        NV_GPU_PERF_PSTATE_ID currentpstate;
```

Comment:
Lets move this before before printing the different clocks, since they are a result of the current p-state.  
Please also change the order above.